config/samples/hydra_v1alpha1_oauth2client_user_credentials.yaml

```diff
@@ -37,3 +37,6 @@ spec:
     endpoint: /clients
     forwardedProto: https
   tokenEndpointAuthMethod: client_secret_basic
+  metadata:
+    property1: 1
+    propetry2: "2"
```

Comment:
typo

Reply:
actually it is irrelevant, since any name can be property, but I will correct it.

thanks